src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -957,9 +957,51 @@ In the next section, we'll walk you through configuring your notification settin
 
     [{% asset slack-alert-message.png %}]({% asset slack-alert-message.png @path %})
     
-#### Deleting the legacy Slack integration
+#### Deleting the legacy project-based Slack integration
```

Comment:
I updated this part because I thought it would be confusing for people if they didn't know what the Slack plugin was, and I didn't want folks to think the should delete the integration and reinstall. They should be going through the upgrade flow